match/lib/match/nuke.rb

```diff
@@ -89,11 +104,11 @@ def prepare_list
         self.profiles += profile_type(prov_type).all
       end
 
-      certs = Dir[File.join(params[:workspace], "**", cert_type.to_s, "*.cer")]
-      keys = Dir[File.join(params[:workspace], "**", cert_type.to_s, "*.p12")]
+      certs = Dir[File.join(self.storage.working_directory, "**", cert_type.to_s, "*.cer")]
+      keys = Dir[File.join(self.storage.working_directory, "**", cert_type.to_s, "*.p12")]
```

Comment:
I think it'd be more elegant to provide this as a method on the storage interface, e.g.

```
self.storage.find_all(File.join(cert_type.to_s, "*.cer")) 
```